### ISSUE TITLE
Kata order not apparent, but it is important

### ISSUE THREAD

**Issue by erquhart** [tddbin/es6katas.org#10]
I haven't completed the Array API kata yet, but It's already assumed knowledge of a few es6 concepts covered in other katas. Since the katas are my first dive into es6, I had to go learn those things to complete the work.

Have you considered setting an obvious order for these, so that learning is progressive? As I said, I'm working on the Array API kata, but I'm not sure if it's at the top left because it's oldest or newest - there's no clear starting point. Running into concepts covered in other katas makes me feel like I'm going out of order.

**Comment by tddbin** [comment]
the idea is great and in here
https://github.com/tddbin/katas/blob/master/katas/es6/language/__raw-metadata__.js
you
can see this all, its just not embedded in the site yet :(
The suggestion you made is great, to offer kinda progressive learning set,
where you are lead through the katas.
input welcome

--
Kind regards / Saludos / Mit freundlichen Grüßen
Wolfram Kriesing, uxebu Co-Founder

kriesing@uxebu.com, mobile: +49 174 300 4595

ECMAScript6+React.js workshop http://meetu.ps/2Jds4D
Learn ES6, one kata a day http://es6katas.org

uxebu GmbH
Fraunhofer Straße 31e
86167 Augsburg

Amtsgericht Augsburg, Handelsregister HRB 28613
Directors: W. Kriesing, T. v. Klipstein

**Comment by erquhart** [comment]
Okay, cool. Is the design from issue #1 something you would like to see implemented? I can work on a PR if so. The current look does have this cool, raw feel to it - is likely alter the design a bit so as not to lose that.

**Comment by tddbin** [comment]
Tbh I took inspiration from #1 but that's it.
Am 29.06.2015 14:52 schrieb "Shawn Erquhart" <notifications@github.com>:

**Comment by erquhart** [comment]
So you're saying you like the design as is, correct?

**Comment by tddbin** [comment]
Mmmh, I don't really. I would be open to more modern looks
Am 29.06.2015 17:21 schrieb "Shawn Erquhart" <notifications@github.com>:

**Comment by tddbin** [comment]
Esp enabling some things like more info about each kata group or maybe even
different kind of views. Eg a view where you can work off katas simplest to
hardest or showing their dependencies
Am 29.06.2015 17:29 schrieb "Wolfram Kriesing" <kriesing@uxebu.com>:

**Comment by erquhart** [comment]
Makes sense. Can you get a hold of the psd from #1? Could be used as a base.

**Comment by tddbin** [comment]
Not sure if #1 works when the number of katas grows ...
Ping get directly to get the PSD,I don't have it either
Am 29.06.2015 17:36 schrieb "Shawn Erquhart" <notifications@github.com>:

**Comment by erquhart** [comment]
I actually think it will work just as well as the current setup - the section containing the katas will simply expand. At a glance I would reduce the size of the illustration and potentially name the sections. Prerequisites for individual katas could be revealed by expanding on hover.

@MadameLeRenoir, was there a psd for your design?

**Comment by MadameLeRenoir** [comment]
Sure got a Pixelmator file which you can import. I thought the issue for a redesign was already closed?

**Comment by erquhart** [comment]
@tddbin (Wolfram I believe?) is open to it, see comment string above.

**Comment by tddbin** [comment]
Let the inspiration flow ...
Wolfram
Am 29.06.2015 19:06 schrieb "Shawn Erquhart" <notifications@github.com>:

**Comment by erquhart** [comment]
@MadameLeRenoir Pixelmator file would work great :)

**Comment by nerdbeere** [comment]
While I'm certainly not a designer I also wanted to give some input about the layout (it's not meant as a design proposal!).

![es6katas-layout](https://cloud.githubusercontent.com/assets/423218/8427388/135d5982-1f17-11e5-9e24-362f56c496c8.png)

**Comment by tddbin** [comment]
that shows all the stuff we need. i like the idea!

--
Kind regards / Saludos / Mit freundlichen Grüßen
Wolfram Kriesing, uxebu Co-Founder

kriesing@uxebu.com, mobile: +49 174 300 4595

ECMAScript6+React.js workshop http://meetu.ps/2Jds4D
Learn ES6, one kata a day http://es6katas.org

uxebu GmbH
Fraunhofer Straße 31e
86167 Augsburg

Amtsgericht Augsburg, Handelsregister HRB 28613
Directors: W. Kriesing, T. v. Klipstein

**Comment by nerdbeere** [comment]
Would be awesome if someone comes up with a design based on this layout.

**Comment by erquhart** [comment]
Awesome concept. Considering the utilitarian nature of the project, I'd say this is pretty close already. Just add color, typography and branding. The real work will be tracking things (could use local storage, just easy to lose your data), gathering documentation links, and creating examples. This also shifts the site from static single page to single page app.

**Comment by wolframkriesing** [comment]
I am back to this idea too, because I somehow feel I want to strengthen the learning aspect of the katas and try to lead people a bit more, if there is still interest I would be curious to implement some cool design. Please come forward. I just can't design at all :)
